modules/guardduty-member-account/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "guardduty_member_detector" {
+  description = "GuardDuty member account detector"
+  value       = aws_guardduty_detector.this
+}
+
+output "guardduty_member" {
```

Comment:
```suggestion
output "member" {
```

Reply:
Removed GuardDuty from all of the output names.